pnpm-workspace.yaml

```diff
@@ -3,3 +3,5 @@ packages:
   - "cli"
   - "packages/*"
   - "benchmark"
+  # Intentionally exclude the `npm` `turbo` package from the workspaces.
```

Comment:
This is not "new" so-to-say. `turbo` only became a package very-recently, but because of how the native dependencies are generated this getting `install`ed results in significant added complexity.